cli/cmd/encore/db.go

```diff
@@ -182,6 +182,9 @@ var dbConnURICmd = &cobra.Command{
 		dbName := ""
 		if len(args) > 0 {
 			dbName = args[0]
+			if _, err := os.Stat(filepath.Join(appRoot, dbName)); err != nil {
```

Comment:
It's possible for a service directory to exist as a subdirectory and not necessarily live directly in the application root.   

But possibly it would be better if we had the daemon return a dedicated status code if the service does not exist, and handle this in the error check on line 210 instead.

Reply:
Thanks @miles170. You're right to think we shouldn't modify generated code. However what you see is the generated code for the GRPC connection. The server-side implementation is here: https://github.com/encoredev/encore/blob/main/cli/daemon/db.go#L22 (which is not generated code)